Jenkinsfile

```diff
@@ -158,13 +158,16 @@ pipeline {
           }
         }
       }
-    }
+    }*/
     stage('Release') {
       agent { label 'docker' }
       options { skipDefaultCheckout() }
       environment {
         GITHUB_TOKEN    = credentials('rv-jenkins')
         KEVM_RELEASE_ID = '1.0.0'
+        PACKAGE = 'kevm'
+        VERSION = '1.0.0'
+        ROOT_URL = 'https://github.com/kframework/evm-semantics/releases/download/nightly'
```

Comment:
Is there a reason for `VERSION` separate from `KEVM_RELEASE_ID`?

Reply:
It doesn't seem to be used directly anywhere here.

Reply:
This is used by the brew scripts that we are invoking that are part of the kframework package. They are used to control the CI so that it will test the correct commit on Mac OS without deploying it first.